assets/a1111webui.js

```diff
@@ -55,16 +54,21 @@ function formatMetadata(valSet) {
         + formatMet('Eta', valSet['eta'])
         + formatMet('Clip skip', valSet['clipskip'], '1')
         // ENSD
-        ;
+        + '';
```

Comment:
huh?

Reply:
A: it's fine as it was
B: `// ENSD` is meant to be inside of the block, before the semicolon, as it is a placeholder indicating a value that is not currently being handled there.